apps/site/site.json

```
@@ -28,9 +28,9 @@
   ],
   "websiteBanners": {
     "index": {
-      "startDate": "2025-01-14T03:00:00.000Z",
-      "endDate": "2025-01-21T03:00:00.000Z",
-      "text": "New security releases to be made available Tuesday, January 21, 2025",
+      "startDate": "2025-01-21T03:00:00.000Z",
+      "endDate": "2025-01-28T03:00:00.000Z",
+      "text": "January Security Release is available",
```

```suggestion
      "text": "January Security Releases are available",
```
I would think this should be plural given the multiple lines released?

---

I'm sorry @MattIPv4 I have resolved it locally but I forgot to push it